docs/video/obs/output.md

```diff
@@ -40,38 +40,56 @@ It's recommended to use the [latest version of OBS](https://github.com/obsprojec
 
     === ":simple-nvidia: (NVIDIA) NVENC"
 
-        !!! abstract "High FPS recording configuration"
+        !!! abstract "Note for File Size Efficiency"
 
-            If you want to record in 120+FPS you're advised to configure the following:
+            The settings provided here are specifically for high framerate recoring, which prefer performance over low file sizes. If file sizes are an issue, consider making the following changes:
 
-            * <div class="annotate" markdown>`Preset`: P1 - P4(1)</div>
+            * <div class="annotate" markdown>`Preset`: P2 - P7(1)</div>
 
-                1. OBS describes low Presets as "Lower Quality" in the context of <u>streaming with CBR in bandwidth-limited scenarios</u>, if used for recording it'll be much more performant to record at the cost of bigger file sizes.
+                1. This setting does not impact quality when using CQP.
```

Comment:
why omit all that important detail to be more implicit about the fact it's referring to the quality of the efficiency?